specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -0,0 +1,2958 @@
+            "pattern": "^[a-z0-9_-]*$"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The requested test run details",
+            "schema": {
+              "$ref": "#/definitions/TestRunModel20221101"
+            }
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a test run": {
+            "$ref": "./examples/GetTestRun.json"
+          }
+        }
+      }
+    },
+    "/testruns/{testRunId}/files/{fileId}": {
+      "get": {
+        "tags": [
+          "TestRun"
+        ],
+        "summary": "Get test run file by file name.",
+        "operationId": "TestRun_GetFile",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "testRunId",
+            "description": "Unique name for the load test run, must contain only lower-case alphabetic, numeric, underscore or hyphen characters.",
+            "required": true,
+            "type": "string",
+            "maxLength": 50,
+            "minLength": 2,
+            "pattern": "^[a-z0-9_-]*$"
+          },
+          {
+            "in": "path",
+            "name": "fileId",
+            "description": "Unique name for test run file, must contain only lower-case alphabetic, numeric, underscore or hyphen characters.",
+            "required": true,
+            "type": "string",
+            "maxLength": 50,
+            "minLength": 2,
+            "pattern": "^[a-z0-9_-]*$"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The URL for the requested test run file",
+            "schema": {
+              "$ref": "#/definitions/FileUrl20221101"
+            }
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a test run file by an unique name": {
+            "$ref": "./examples/GetTestRunFile.json"
+          }
+        }
+      }
+    },
+    "/testruns/sortAndFilter": {
```

Comment:
This is a standard list operation on the testruns collection so it should be a GET on "/testruns"

Reply:
changed